judge/models/submission.py

```diff
@@ -113,8 +113,8 @@ def short_status(self):
     def long_status(self):
         return Submission.USER_DISPLAY_CODES.get(self.short_status, '')
 
-    def judge(self, rejudge=False, batch_rejudge=False):
-        judge_submission(self, rejudge, batch_rejudge)
+    def judge(self, *args, **kwargs):
+        judge_submission(self, *args, **kwargs)
```

Comment:
Pretty sure this'll die if just calling `.judge()`, since `rejudge` is not a kwarg in `judge_submission` (I guess it could be?)

Reply:
`rejudge` is an arg of `judge_submission`, so I think this will work.

Reply:
It's an arg that'll not be passed if you're just calling `.judge()`, not `.judge(True)`. Sounds like you'd just get an incorrect number of parameters passed error.